AdminGuide.md

```diff
@@ -0,0 +1,156 @@
+# Admin Guide for WikiEduDashboardTools
+
+This guide provides detailed instructions for setting up, managing, and troubleshooting WikiEduDashboardTools on Toolforge and Cloud VPS.
+
+## Table of Contents
+1. [Setup Instructions](#setup-instructions)
+   - [Toolforge Setup](#toolforge-setup)
+   - [Cloud VPS Setup](#cloud-vps-setup)
+2. [Managing the System](#managing-the-system)
+   - [Toolforge](#toolforge)
+        - [Starting, Stopping, and Restarting Services](#starting-stopping-and-restarting-services)
+        - [Monitoring and Logs](#monitoring-and-logs)
+        - [Some useful toolforge commands](#some-useful-toolforge-commands)
+   - [Cloud VPS](#cloud-vps)
+        - [Starting, Stopping, and Restarting Instances](#starting-stopping-and-restarting-instances)
+        - [Monitoring](#monitoring)
+        - [Some useful commands](#some-useful-commands)
+3. [Troubleshooting](#troubleshooting)
+    - [Web Server Issues](#web-server-issues)
+    - [Database Issues](#database-issues)
+    - [Data Dumps and Recovery](#data-dumps-and-recovery)
+4. [More Resources](#more-resources)
+
+
+## Setup Instructions
+
+### Toolforge Setup
+1. Clone the repository in the tool's home directory:
+   ```bash
+   git clone https://github.com/WikiEducationFoundation/WikiEduDashboardTools.git
+   ```
+2. Start the web server:
+   ```bash
+   webservice start
+   ```
+
+### Cloud VPS Setup
+1. Create a Debian server with the `web` security group.
+2. Add DNS for an external URL (like dashboard-replica-endpoint.wmcloud.org)
+3. Install required packages:
+   ```bash
+   sudo apt install apache2 php libapache2-mod-php php-mysql
+   ```
+4. Clone the repository into `/var/www/`:
+   ```bash
+   git clone https://github.com/WikiEducationFoundation/WikiEduDashboardTools.git /var/www/WikiEduDashboardTools
+   ```
+4. Configure the web server:
+
+   - Disable the default site and add a new site configuration file (`dashboard-tools.conf`):
+    ``` 
+        <VirtualHost *:80>
+            ServerName dashboard-replica-endpoint.wmcloud.org
+            ServerAdmin sage@wikiedu.org   
+            DocumentRoot /var/www/WikiEduDashboardTools/public_html
+            ErrorLog ${APACHE_LOG_DIR}/error.log
+            CustomLog ${APACHE_LOG_DIR}/access.log combined
+        </VirtualHost> 
+    ```
+   - Increase the PHP memory_limit in `/etc/php/7.4/apache2/php.ini` or a similar location (to 4096M):
+     ```ini
+     memory_limit = 4096M
+     ```
+     - This ensures that queries that return very large amounts of data won't run PHP out of memory while converting query results to JSON.
+   - Enable the new site, and visit /index.php
+
+5. Add Replica database credentials:
+   - Copy `replica.my.cnf` from a Toolforge tool account into the tool's root directory.
+6. Test a query endpoint, for example: 
+   - `/revisions.php?lang=en&project=wikipedia&usernames[]=Ragesoss&start=20140101003430&end=20171231003430`
+
+
+## Managing the System
+
+### Toolforge
```

Comment:
I don't think we need to collect any basic Toolforge/CloudVPS commands here unless they have a specific known use case in relation to maintain the dashboard. This guide should assume that someone trying to use it has a general proficiency with the Toolforge environment, and just doesn't know things that are specific to how the dashboard is run on Toolforge and CloudVPS

Reply:
Okay, all noted!